frontend/src/app/patients/EditPatient.tsx

```diff
@@ -303,84 +387,8 @@ const EditPatient = (props: Props) => {
               }}
               patientId={props.patientId}
               savePerson={savePerson}
-              getHeader={(person, onSave, formChanged) => (
```

Comment:
i had a physical reaction reading the way we nested this logic, ty for pulling it out!